.github/workflows/release-homebrew.yaml

```diff
@@ -1,34 +1,18 @@
 name: "release-homebrew"
 on:
   release:
-    types: [published]
+    types: [released]
```

Comment:
Does this trigger on `pre-release` to `release` instead of publishing a draft to `pre-release`?

Reply:
So right now, this change will mean the Homebrew PR only gets created when doing a `release`/not-a-draft/not-a-pre-release.